### ISSUE TITLE
Global chart setup in analysis tab 

### ISSUE THREAD

**Issue by schneiderst** [AquaCalculator/Support#1]
As a user I want to be able to setup global parameters for all displayed charts in tab analysis.
As a user I want to use global setup to select number of days and zoom for all displayed charts.

**Comment by mimorocks** [comment]
the point is to do this for all analysis at one and not on a per element base?

**Comment by schneiderst** [comment]
correct, it's nice to have the option for each chart but most cases you would compare charts on the same time scale
